tests/bears/BearTest.py

```diff
@@ -157,6 +157,13 @@ class aspectsFixOnlyTestBear(Bear, aspects={
     pass
 
 
+class aspectsGeneralTestBear(Bear, aspects={
```

Comment:
This should go in test_bears dir , after https://github.com/coala/coala/pull/5651

Reply:
I see, I'll keep it here for now else the tests won't pass. Would rebase as soon as you get the issue solved :+1:

Reply:
It is ready

Reply:
Shifted `LANGUAGE_NAMES` and `LANGUAGE_COUNT` to `TestUtilities.py` as you did in  #5651.

Reply:
This Bear needs to be a separate module in our tests/test_bears/ directory